### ISSUE TITLE
Hyperlinks in Master Slides and Slides

### ISSUE THREAD

**Issue by Nephtuk** [gitbrent/PptxGenJS#257]
Hey @gitbrent,

I have some issues with adding hyperlinks to my slides. 
Adding hyperlinks as an option to addImage() does not seem to work and adding it as an option to slide.addText() seems to corrupt the ppt completely.
These are just minor problems. My biggest issue is, that I am not able to add a hyperlink to a Master Slide.
I tried to add the hyperlink with the defineSlideMaster function: 
`objects: [{ 'text':  
        {text: "This is a text.",
	    options: {y: 5.35, w: '95%', font_size: 8, font_face: 'Arial', color: '14528C',
                  hyperlink: {url: 'https://github.com/gitbrent/pptxgenjs'}
            }
         }
      }
]});`

This, too, corrupts my ppt. I am not sure, if I applied the functions incorrectly or hyperlinks in Master Slides are not supported yet. I hope I could describe the issue clearly.

PowerPoint-Version: Microsoft PowerPoint 2010

Thank you in advance!

**Comment by gitbrent** [comment]
Hi @Nephtuk,

Does the demo image hyperlink work for you?
`examples/pptxgenjs-demo.html`

![screen shot 2017-12-14 at 19 02 17](https://user-images.githubusercontent.com/7218970/34021826-7221cd28-e101-11e7-806c-031498c3186c.png)

**Comment by Nephtuk** [comment]
Hello @gitbrent ,

unfortunately neither the master slide nor a normal slide does work with that link. The PPT is still getting corrupted. 
Could there be a problem with the function createHyperlinkRels in line 1951? If I do 

`slide.addText("Link", {x: 0, y: 0, hyperlink: {url: 'https://github.com/gitbrent/pptxgenjs/examples/pptxgenjs-demo.html'}})`

intext is only a String and the function returns immediately. But if it is a Object, it pushes the Hyperlink-Object. I tried it with a textobject that looked like this: 

`{text: 'Link', options: {x:0, y:0, hyperlink: {url: 'https://github.com/gitbrent/pptxgenjs/examples/pptxgenjs-demo.html''}}}`

This goes through as a textobject, but with this Syntax in slide.addText the PPT will be corrupted again. I tested this with a normal slide.

In the master slide I added the hyperlink like this: 

`object: [{ 'text': {text: "Link", options: {x: 0, y: 0, hyperlink: {url: 'https://github.com/gitbrent/pptxgenjs/examples/pptxgenjs-demo.html'}}}}]`

intext is here too, just the String 'Link' instead of the whole textobject. At this point I am a little bit confused. I am not sure, maybe I did not understand the function correctly, but I think here could be a mistake.
